train.py

```diff
@@ -0,0 +1,228 @@
+import uuid
+import itertools
+from typing import Callable
+
+import hydra
+import torch.optim
+import pandas as pd
+import torch.nn as nn
+from tqdm import tqdm
+from fortepyan import MidiPiece
+from sklearn.metrics import f1_score
+from torch.utils.data import DataLoader
+from omegaconf import OmegaConf, DictConfig
+
+import wandb
+from model import PitchSeqNN
+from utils import test_model
+from data.dataset import BagOfPitches
+
+
+@hydra.main(version_base=None, config_path="conf", config_name="config")
+def main(cfg: DictConfig):
+    classnames = ["Ludwig van Beethoven", "Franz Liszt"]
+    model = run_experiment(cfg, classnames=classnames)
+    test_data = BagOfPitches(split="test", selected_composers=classnames)
+    test_dataloader = DataLoader(test_data, batch_size=cfg.hyperparameters.batch_size, shuffle=True)
+
+    # test model and get all wrongly predicted samples
+    wrongly_predicted = wrong_preds(model, test_dataloader)
```

Comment:
```suggestion
    bad_predictions = wrong_preds(model, test_dataloader)
```

Using adverbs for variable names should be avoided. When naming variables, you typically want nouns or adjectives because variables represent entities or states, not actions or qualities of actions.